alsa.te

```diff
@@ -30,15 +36,13 @@ files_type(alsa_var_lib_t)
 type alsa_var_lock_t;
 files_lock_file(alsa_var_lock_t)
 
-type alsa_home_t;
-userdom_user_home_content(alsa_home_t)
-
 ########################################
 #
 # Local policy
 #
 
-allow alsa_t self:capability { dac_read_search dac_override setgid setuid ipc_owner };
+# kill : kill itself
+allow alsa_t self:capability { dac_read_search dac_override kill setgid setuid ipc_owner };
```

Comment:
You're right, its killing pulseaudio on shutdown
```
type=PROCTITLE msg=audit(12/01/17 16:00:58.422:1928) : proctitle=/usr/sbin/alsactl -E HOME=/run/alsa store 
type=OBJ_PID msg=audit(12/01/17 16:00:58.422:1928) : opid=1699 oauid=christian ouid=christian oses=3 obj=unconfined_u:unconfined_r:unconfined_t:s0 ocomm=pulseaudio 
type=SYSCALL msg=audit(12/01/17 16:00:58.422:1928) : arch=x86_64 syscall=kill success=no exit=EACCES(Permission denied) a0=0x6a3 a1=SIG0 a2=0xbeefcafe a3=0x1 items=0 ppid=1 pid=29683 auid=unset uid=root gid=root euid=root suid=root fsuid=root egid=root sgid=root fsgid=root tty=(none) ses=unset comm=alsactl exe=/usr/sbin/alsactl subj=system_u:system_r:alsa_t:s0 key=(null) 
type=AVC msg=audit(12/01/17 16:00:58.422:1928) : avc:  granted  { kill } for  pid=29683 comm=alsactl capability=kill  scontext=system_u:system_r:alsa_t:s0 tcontext=system_u:system_r:alsa_t:s0 tclass=capability
```
Maybe dontaudit'ing it is appropriate?